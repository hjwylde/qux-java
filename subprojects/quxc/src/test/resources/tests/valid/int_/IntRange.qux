void f([int] is):
    for i in is:
        print i

void main([str] args):
    f(0..1)
    f(0..0)
    f(5..10)
    f(0..3 - [1])
    print 0..2

