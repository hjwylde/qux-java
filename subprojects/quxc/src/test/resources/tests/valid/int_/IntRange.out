0
5
6
7
8
9
0
2
[0, 1]
